### ISSUE TITLE
Update dialog shows for Flatpak for the new Vesktop version

### ISSUE THREAD

**Issue by Mv0sKff** [Vencord/Vesktop#441]
**Describe the bug**

When installing Vesktop via flatpak it will show the "Update Available" dialog (Screenshot) when an update is available, which is not responsible for updating the flatpak version.

**To Reproduce**

1. Downgrade Vesktop
2. Run Vesktop
```
sudo flatpak update --commit=70a623e0e430953bbd39aac618662b4009459692c82c335b1131f412e4e9b047 dev.vencord.Vesktop
flatpak run dev.vencord.Vesktop
```

**Expected behavior**

Ignore Vesktop updates, since they are managed by flatpak.

**Fix**
If this issue is relevant, I would consider fixing it in a PR myself.

**Screenshots**

![image](https://github.com/Vencord/Vesktop/assets/47915396/38f137ea-25ef-4e2e-9f16-836f2d2e8935)

**Desktop**
```
OS: Fedora Linux 39 (Workstation Edition) x86_64
Desktop: GNOME 45.4 Wayland
```

**Comment by Redo11** [comment]
It would be nice, if it would auto-update the flatpak with a single click and then restart the app.

**Comment by Vendicated** [comment]
resolved in 1.5.3
